### PR TITLE
Position+Attitude complex message

### PR DESCRIPTION
**Pull request by EShamaev**
I have added baro height and linear_acceleration to the proposed message

### CODE REVIEW

**Comment by EShamaev** [comment]
Added distance to ground if aircraft is equipped with LIDAR or radalt

File: uavcan/equipment/ahrs/1002.PositionAttitude.uavcan [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#
+
+uavcan.Timestamp timestamp
+
+float64 longitude                   # angular degree
+float64 latitude                    # angular degree
+float32 height_ellipsoid            # meter
+float32 height_msl                  # meter
+float32 height_baro                 # meter
+float32 ground_distance             # meter (with radar altimeter or LIDAR)
```
Comment (pavel-kirienko) [inline uavcan/equipment/ahrs/1002.PositionAttitude.uavcan]: Shall we rename `ground_distance` into `height_agl` for consistency?

Also, that's a lot of heights, but I don't have any better ideas, so it seems that we're going to have to live with this now.

Comment (EShamaev) [inline uavcan/equipment/ahrs/1002.PositionAttitude.uavcan]: Yes, in aviation baro is used a lot but it also changes without "real" connection to any physical altitude. For example when plane is in vicinity of airplane zero baro is set from tower, when in free flight, all baro's zeros on all planes are set to same value.
But this value is really needed to be reported.
For height_agl this one is strictly based on ground and follow terrain but is very good for photo missions to store.

File: uavcan/equipment/ahrs/1002.PositionAttitude.uavcan [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#   height (worse)  [meter^2]
+#   roll angle      [radian^2]
+#   pitch angle     [radian^2]
+#   yaw angle       [radian^2]
+#
+float16 pose_covariance[<=36]
+
+float32[3] linear_velocity_ned      # meter/second
+float16[3] angular_velocity         # rad/sec
+float16[3] linear_acceleration      # m/s^2
```
Comment (EShamaev) [inline uavcan/equipment/ahrs/1002.PositionAttitude.uavcan]: I put it here not for fusion reasons, but as you stated later as a filtered solution to be used right away for logging simple things like uncoordinated turn condition, etc.

**Comment by pavel-kirienko** [comment]
Sorry to keep you hanging Eugene. I suggest to move this message away from the AHRS namespace into a new second level namespace `uavcan.navigation.`. Do you think that makes sense? I am not comfortable with keeping this definition in the equipment namespace because it is not pertaining to any specific kind of equipment.

**Comment by EShamaev** [comment]
Fully agree. Will do.

**Comment by pavel-kirienko** [comment]
Hey Eugene, it is just a reminder that we're one commit away from merging this.

**Comment by EShamaev** [comment]
Moved and added QNH parameter as an essential information on baro altitude calculation

**Comment by pavel-kirienko** [comment]
Superseded by https://github.com/UAVCAN/dsdl/pull/23. I had to open a new pull request because I can't push to the fork owned by @EShamaev.

**Pull request closed without merging**